src/bindings/python/flux/util.py

```diff
@@ -218,7 +218,13 @@ def __call__(self, main_func):
             exit_code = ex
         except Exception as ex:  # pylint: disable=broad-except
             exit_code = 1
-            self.logger.error(str(ex))
+            # Prefer '{strerror}[: {filename}' error message over default
```

Comment:
is that `[` in here a typo?  Could just tack on top of #4934 if it is.

Reply:
Yes. Good catch